pyclibrary/c_parser.py

```diff
@@ -407,15 +153,16 @@ class CParser(object):
     def __init__(self, files=None, copy_from=None, replace=None,
                  process_all=True, cache=None, **kwargs):
 
+        self.clib_intf = c_model.CLibInterface()
+        if copy_from is not None:
+            self.clib_intf.include(copy_from)
+        self.macro_vals = {}
+
         if not self._init:
             logger.warning('Automatic initialisation based on OS detection')
             from .init import auto_init
             auto_init()
 
-        # Holds all definitions
-        self.defs = {}
-        # Holds definitions grouped by the file they came from
-        self.file_defs = {}
         # Description of the struct packing rules as defined by #pragma pack
```

Comment:
We will have to determine how we pass that info from the preprocessor to the parser. Any idea ?